internal/attest/attest.go

```diff
@@ -68,14 +71,14 @@ func NewAttestor(repo *git.Repository, sv *sign.SignerVerifier, rekorFn rekorUpl
 }
 
 // WriteFile is a convenience wrapper around WriteAttestation that takes in a filepath rather than an io.Reader.
-func (a *Attestor) WriteFile(ctx context.Context, refName string, sha plumbing.Hash, path, attType string) (plumbing.Hash, error) {
+func (a *Attestor) WriteFile(ctx context.Context, refName string, sha plumbing.Hash, path, attType string, config *gitsignconfig.Config) (plumbing.Hash, error) {
```

Comment:
question: should we have the git config in the Attestor struct?

Reply:
This was my original idea because it would minimize the amount of times I pass around the pointer to the config, but I didn't know how comfortable everyone would feel about me changing the base Attestor struct. I'll added it now.